igcollect/zookeeper.py

```diff
@@ -39,19 +39,19 @@ def main():
             print(template.format(key, value))
 
 
-def netcat(hostname, port, content):
-    mntr = ''
+def netcat(hostname: str, port: int, content: str) -> str:
+    data = b''
     conn = socket(AF_INET, SOCK_STREAM)
     conn.connect((hostname, port))
-    conn.sendall(content)
+    conn.sendall(content.encode())
     conn.shutdown(SHUT_WR)
     while True:
-        data = conn.recv(1024)
-        if data == '':
+        answer = conn.recv(1024)
+        if answer == b'':
```

Comment:
Do we need the explicit comparison or could we also go with:
```suggestion
        if not answer:
```
?